resources/js/components/forms/PhoneInput.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div :class="wrapperClass">
+    <label v-if="label" :for="id ? id : name" :class="labelClasses">
+      {{ label }}
+      <span v-if="required" class="text-red-500 required-dot">*</span>
+    </label>
+    <small v-if="help && helpPosition === 'above_input'" :class="theme.default.help" class="flex mb-1">
+      <slot name="help"><span class="field-help" v-html="help" /></slot>
+    </small>
+
+    <div class="flex">
```

Comment:
As for the examples, it needs to be styled as a single input (see how we did date range input for instance)

Reply:
Not 2 distincs fields like you have:
![image](https://github.com/JhumanJ/OpnForm/assets/11312432/ee431f78-88af-4ec8-8341-391c103016ab)